vagrant/config/docker/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
ok. I'm thinking of maintain a template config.json, and modify based on this template for diff purpose.
otherwise, it's hard to keep all of them in sync.  example: RAC-4981